### ISSUE TITLE
Illegal char <:> exception on Windows

### ISSUE THREAD

**Issue by milelo** [nextjournal/clerk-demo#8]
This is still an issue on clerk-demo sha: 8cbed3763289ebe8d5616a0e2896dc5a1b3c0059
As reported on Slack nextjournal
I've got a small Clerk project I created already running ok on my system.
I'm trying to start:
https://github.com/nextjournal/clerk-demo
from a Windows terminal using "clj" (CLI version 1.10.3.933).
I get a startup exception (updated):
Caused by: org.graalvm.polyglot.PolyglotException: java.nio.file.InvalidPathException: Illegal char <:> at index 4: file:/C:/Users/mail/.m2/repository/io/github/nextjournal/markdown/0.1.36/markdown-0.1.36.jar!/js/markdown.mjs

mkvlr  initial response:
this looks like [https://github.com/nextjournal/viewers/issues/29](https://github.com/nextjournal/viewers/issues/29) 
This has now been closed but this clerk-demo issue still persists.

**Comment by milelo** [comment]
I'm using the current clerk-demo so "23a0d84f0566e0db74dc30b65cebaa62b53a9c42"
user=> (System/getProperty "java.vendor")
"AdoptOpenJDK"
user=> (System/getProperty "java.version")
"1.8.0_292"

**Comment by jackrusher** [comment]
@milelo Can you check if this is still a problem? I don't have access to a Windows environment.

**Comment by milelo** [comment]
The reported problem is now resolved. Thanks, Jack.

Errors occur during build but I kept retrying and eventually the build completed.

The file watcher reports an error.

These notebooks render without the file watcher:

  (clerk/show! "notebooks/data_science.clj")
  (clerk/show! "notebooks/sicmutils.clj")
  (clerk/show! "notebooks/rule_30.clj")
  (clerk/show! "notebooks/images.clj")

These have errors and won't render:
  (clerk/show! "notebooks/introduction.clj")
  (clerk/show! "notebooks/semantic.clj")

I can formally report the other problems when I get the opportunity.
